I see that this error keeps occurring.

> [rtmlaunch] [ERROR] Could not Connect ( /localhost:15005/HiroNX(Robot)0.rtc:CAMERA_HEAD_R , /localhost:15005/ImageSensorROSBridge_HeadRight.rtc:timedImage ):  'NoneType' object has no attribute 'connections'

```
$ rtls localhost:15005/                     
StateHolderServiceROSBridge.rtc        fk.rtc
DataLoggerServiceROSBridge.rtc         longfloor(Robot)0.rtc
ImageSensorROSBridge_HandLeft.rtc      sh.rtc
HiroNX(Robot)0.rtc                     ImageSensorROSBridge_HeadRight.rtc
seq.rtc                                HrpsysJointTrajectoryBridge0.rtc
log.rtc                                HGcontroller0.rtc
sc.rtc                                 ModelLoader
el.rtc                                 CollisionDetector0.rtc
ImageSensorROSBridge_HandRight.rtc     ImageSensorROSBridge_HeadLeft.rtc
HrpsysSeqStateROSBridge0.rtc           SequencePlayerServiceROSBridge.rtc
ForwardKinematicsServiceROSBridge.rtc  tork-kudu1.host_cxt/
ic.rtc                                 rmfo.rtc
```

```
$ rtmlaunch nextage_ros_bridge nextage_ros_bridge_simulation.launch
[rtmlaunch] Start omniNames at port 15005

Thu Jan 19 12:08:04 2017:

Starting omniNames for the first time.
Wrote initial log file.
Read log file successfully.
Root context is IOR:010000002b00000049444c3a6f6d672e6f72672f436f734e616d696e672f4e616d696e67436f6e746578744578743a312e300000010000000000000070000000010102000d0000003139322e3136382e302e313800009d3a0b0000004e616d6553657276696365000300000000000000080000000100000000545441010000001c000000010000000100010001000000010001050901010001000000090101000354544108000000a41c8158010013f6
Checkpointing Phase 1: Prepare.
Checkpointing Phase 2: Commit.
Checkpointing completed.
... logging to /home/rospasta/.ros/log/fcc43aec-de82-11e6-ab69-80fa5b08d1d8/roslaunch-tork-kudu1-5104.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://tork-kudu1:36274/

SUMMARY
========

PARAMETERS
 * /HrpsysSeqStateROSBridge/publish_sensor_transforms: True
 * /collision_state/comp_name: CollisionDetector0
 * /controller_configuration: [{'group_name': '...
 * /diagnostic_aggregator/analyzers/computers/contains: ['HD Temp', 'CPU ...
 * /diagnostic_aggregator/analyzers/computers/path: Computers
 * /diagnostic_aggregator/analyzers/computers/type: diagnostic_aggreg...
 * /diagnostic_aggregator/analyzers/hrpsys/contains: ['hrpEC', 'Emerge...
 * /diagnostic_aggregator/analyzers/hrpsys/path: Hrpsys
 * /diagnostic_aggregator/analyzers/hrpsys/type: diagnostic_aggreg...
 * /diagnostic_aggregator/analyzers/mode/contains: ['Operating Mode'...
 * /diagnostic_aggregator/analyzers/mode/path: Mode
 * /diagnostic_aggregator/analyzers/mode/type: diagnostic_aggreg...
 * /diagnostic_aggregator/analyzers/motor/contains: ['Motor']
 * /diagnostic_aggregator/analyzers/motor/path: Motor
 * /diagnostic_aggregator/analyzers/motor/type: diagnostic_aggreg...
 * /diagnostic_aggregator/base_path:
 * /diagnostic_aggregator/pub_rate: 1.0
 * /hand_left/ImageSensorROSBridge_HandLeft/frame_id: /CAMERA_HAND_L
 * /hand_right/ImageSensorROSBridge_HandRight/frame_id: /CAMERA_HAND_R
 * /left/ImageSensorROSBridge_HeadLeft/frame_id: /CAMERA_HEAD_L
 * /right/ImageSensorROSBridge_HeadRight/frame_id: /CAMERA_HEAD_R
 * /robot_description: <?xml version="1....
 * /rosdistro: indigo
 * /rosversion: 1.11.20
 * /rtmnameserver_port: 15005
 * /rtmnameserver_robotname: HiroNX(Robot)0
 * /use_sim_time: True

NODES
  /left/
    ImageSensorROSBridge_HeadLeft (hrpsys_ros_bridge/ImageSensorROSBridge)
  /hand_left/
    ImageSensorROSBridge_HandLeft (hrpsys_ros_bridge/ImageSensorROSBridge)
  /right/
    ImageSensorROSBridge_HeadRight (hrpsys_ros_bridge/ImageSensorROSBridge)
  /
    CollisionDetectorComp (hrpsys/CollisionDetectorComp)
    DataLoggerServiceROSBridge (hrpsys_ros_bridge/DataLoggerServiceROSBridgeComp)
    ForwardKinematicsServiceROSBridge (hrpsys_ros_bridge/ForwardKinematicsServiceROSBridgeComp)
    HrpsysJointTrajectoryBridge (hrpsys_ros_bridge/HrpsysJointTrajectoryBridge)
    HrpsysSeqStateROSBridge (hrpsys_ros_bridge/HrpsysSeqStateROSBridge)
    SequencePlayerServiceROSBridge (hrpsys_ros_bridge/SequencePlayerServiceROSBridgeComp)
    StateHolderServiceROSBridge (hrpsys_ros_bridge/StateHolderServiceROSBridgeComp)
    collision_state (hrpsys_ros_bridge/collision_state.py)
    diagnostic_aggregator (diagnostic_aggregator/aggregator_node)
    hrpsys (hrpsys/hrpsys-simulator)
    hrpsys_profile (hrpsys_ros_bridge/hrpsys_profile.py)
    hrpsys_py (nextage_ros_bridge/nextage.py)
    hrpsys_ros_diagnostics (hrpsys_ros_bridge/diagnostics.py)
    hrpsys_state_publisher (robot_state_publisher/state_publisher)
    modelloader (openhrp3/openhrp-model-loader)
    rqt_hironxo (rqt_gui/rqt_gui)
    rtmlaunch_collision_detector (openrtm_tools/rtmlaunch.py)
    rtmlaunch_hrpsys_ros_bridge (openrtm_tools/rtmlaunch.py)
    rtmlaunch_nextage_ros_bridge_simulation (openrtm_tools/rtmlaunch.py)
    sensor_ros_bridge_connect (hrpsys_ros_bridge/sensor_ros_bridge_connect.py)
  /hand_right/
    ImageSensorROSBridge_HandRight (hrpsys_ros_bridge/ImageSensorROSBridge)

WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtactivate
WARNING: unrecognized tag rtactivate
WARNING: unrecognized tag rtactivate
WARNING: unrecognized tag rtactivate
WARNING: unrecognized tag rtactivate
WARNING: unrecognized tag rtactivate
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtactivate
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtconnect
WARNING: unrecognized tag rtactivate
WARNING: unrecognized tag rtactivate
WARNING: unrecognized tag rtactivate
WARNING: unrecognized tag rtactivate
auto-starting new master
process[master]: started with pid [5119]
ROS_MASTER_URI=http://localhost:11311

setting /run_id to fcc43aec-de82-11e6-ab69-80fa5b08d1d8
process[rosout-1]: started with pid [5132]
started core service [/rosout]
process[modelloader-2]: started with pid [5149]
ready
process[hrpsys-3]: started with pid [5154]
process[hrpsys_py-4]: started with pid [5155]
process[HrpsysSeqStateROSBridge-5]: started with pid [5156]
process[HrpsysJointTrajectoryBridge-6]: started with pid [5157]
process[hrpsys_state_publisher-7]: started with pid [5165]
process[hrpsys_ros_diagnostics-8]: started with pid [5184]
process[diagnostic_aggregator-9]: started with pid [5206]
process[hrpsys_profile-10]: started with pid [5226]
process[sensor_ros_bridge_connect-11]: started with pid [5236]
process[rtmlaunch_hrpsys_ros_bridge-12]: started with pid [5238]
loading file:///opt/ros/indigo/share/openhrp3/share/OpenHRP-3.1/sample/model/longfloor.wrl
Humanoid node
Joint node WAIST
  Segment node BODY
The model was successfully loaded !
Loading body customizer "/opt/ros/indigo/share/OpenHRP-3.1/customizer/libSampleCustomizer.so" for springJoint
loading file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
Collada Warning: COLLADA warning: The DOM was unable to create an element named copyright at line 10. Probably a schema violation.

process[SequencePlayerServiceROSBridge-13]: started with pid [5245]
process[DataLoggerServiceROSBridge-14]: started with pid [5266]
process[ForwardKinematicsServiceROSBridge-15]: started with pid [5276]
process[StateHolderServiceROSBridge-16]: started with pid [5283]
process[CollisionDetectorComp-17]: started with pid [5323]
loading /home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_description/models/main.wrl
Collada Warning: fill asset info
Collada Warning: Target Node 'visual1/node_joint0_axis0' not found
process[rtmlaunch_collision_detector-18]: started with pid [5327]
Collada Warning: could not find binding for axis: kmodel1/jointsid1000/axis0, axis0
Collada Warning: could not find binding for axis: kmodel1/jointsid1000/axis0, axis0
process[collision_state-19]: started with pid [5338]
Humanoid node
Joint node WAIST
  Segment node WAIST
  Joint node CHEST_JOINT0
    Segment node CHEST_JOINT0_Link
    Joint node HEAD_JOINT0
      Segment node HEAD_JOINT0_Link
      Joint node HEAD_JOINT1
        Segment node HEAD_JOINT1_Link
        VisionSensorCAMERA_HEAD_R
        VisionSensorCAMERA_HEAD_L
    Joint node RARM_JOINT0
      Segment node RARM_JOINT0_Link
      Joint node RARM_JOINT1
        Segment node RARM_JOINT1_Link
        Joint node RARM_JOINT2
          Segment node RARM_JOINT2_Link
          Joint node RARM_JOINT3
            Segment node RARM_JOINT3_Link
            Joint node RARM_JOINT4
              Segment node RARM_JOINT4_Link
              Joint node RARM_JOINT5
                Segment node RARM_JOINT5_Link
                VisionSensorCAMERA_HAND_R
    Joint node LARM_JOINT0
      Segment node LARM_JOINT0_Link
      Joint node LARM_JOINT1
        Segment node LARM_JOINT1_Link
        Joint node LARM_JOINT2
          Segment node LARM_JOINT2_Link
          Joint node LARM_JOINT3
            Segment node LARM_JOINT3_Link
            Joint node LARM_JOINT4
              Segment node LARM_JOINT4_Link
              Joint node LARM_JOINT5
                Segment node LARM_JOINT5_Link
                VisionSensorCAMERA_HAND_L
Collada Warning: joint HEAD_JOINT1 has axis: 0.000000 1.000000 0.000000
Collada Warning: joint HEAD_JOINT0 has axis: 0.000000 0.000000 1.000000
Collada Warning: joint RARM_JOINT5 has axis: 1.000000 0.000000 0.000000
Collada Warning: joint RARM_JOINT4 has axis: 0.000000 1.000000 0.000000
Collada Warning: joint RARM_JOINT3 has axis: 0.000000 0.000000 1.000000
Collada Warning: joint RARM_JOINT2 has axis: 0.000000 1.000000 0.000000
Collada Warning: joint RARM_JOINT1 has axis: 0.000000 1.000000 0.000000
Collada Warning: joint RARM_JOINT0 has axis: 0.000000 0.000000 1.000000
process[rqt_hironxo-20]: started with pid [5347]
Node is inconvertible and removed from the scene graph
Collada Warning: joint LARM_JOINT5 has axis: 1.000000 0.000000 0.000000
Collada Warning: joint LARM_JOINT4 has axis: 0.000000 1.000000 0.000000
Collada Warning: joint LARM_JOINT3 has axis: 0.000000 0.000000 1.000000
Collada Warning: joint LARM_JOINT2 has axis: 0.000000 1.000000 0.000000
Collada Warning: joint LARM_JOINT1 has axis: 0.000000 1.000000 0.000000
Collada Warning: joint LARM_JOINT0 has axis: 0.000000 0.000000 1.000000
Collada Warning: joint CHEST_JOINT0 has axis: 0.000000 0.000000 1.000000
Collada Warning: cannot find instance_actuator for attached sensor HiroNX:WAIST
Node is inconvertible and removed from the scene graph
process[rtmlaunch_nextage_ros_bridge_simulation-21]: started with pid [5351]
Warning: Mass is zero. <Model>HiroNX <Link>CHEST_JOINT0
Warning: Mass is zero. <Model>HiroNX <Link>WAIST
process[right/ImageSensorROSBridge_HeadRight-22]: started with pid [5374]
The model was successfully loaded !
[ERROR] [1484856486.564443584]: Target Node visual1/node_joint0_axis0 NOT found!!!

;;
;; Could not open /dev/console for writing.
;;
open: Permission denied
[CollisionDetector0] onInitialize()
process[left/ImageSensorROSBridge_HeadLeft-23]: started with pid [5399]
cache found for file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
[sensor_ros_bridge_connect.py]  start
configuration ORB with localhost:15005
[sensor_ros_bridge_connect.py]  waitForRTCManagerAndRoboHardware has renamed to waitForRTCManagerAndRoboHardware: Please update your code
process[hand_right/ImageSensorROSBridge_HandRight-24]: started with pid [5412]
The model was successfully loaded !
Loading body customizer "/opt/ros/indigo/share/OpenHRP-3.1/customizer/libSampleCustomizer.so" for springJoint
Loading body customizer "/opt/ros/indigo/share/OpenHRP-3.1/customizer/libSampleCustomizer.so" for springJoint
cache found for /home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_description/models/main.wrl
duplicated sensor Id is specified(id = 0, name = CAMERA_HEAD_R)
duplicated sensor Id is specified(id = 1, name = CAMERA_HEAD_L)
duplicated sensor Id is specified(id = 2, name = CAMERA_HAND_R)
duplicated sensor Id is specified(id = 3, name = CAMERA_HAND_L)
[ INFO] [1484856486.632075964]: [HrpsysJointTrajectoryBridge] @Initilize name : HrpsysJointTrajectoryBridge0 done
process[hand_left/ImageSensorROSBridge_HandLeft-25]: started with pid [5418]
[CollisionDetector0] prop[collision_pair] ->RARM_JOINT2:LARM_JOINT2 RARM_JOINT3:LARM_JOINT3 RARM_JOINT4:LARM_JOINT4 RARM_JOINT5:LARM_JOINT5
[CollisionDetector0] check collisions between RARM_JOINT2 and LARM_JOINT2
[CollisionDetector0] check collisions between RARM_JOINT3 and LARM_JOINT3
[CollisionDetector0] check collisions between RARM_JOINT4 and LARM_JOINT4
[CollisionDetector0] check collisions between RARM_JOINT5 and LARM_JOINT5
[rtmlaunch] starting...  /opt/ros/indigo/share/hrpsys_ros_bridge/launch/hrpsys_ros_bridge.launch
[rtmlaunch] starting...  /opt/ros/indigo/share/hrpsys_ros_bridge/launch/collision_detector.launch
[rtmlaunch] RTCTREE_NAMESERVERS localhost:15005 localhost:15005
[rtmlaunch] SIMULATOR_NAME HiroNX(Robot)0
[rtmlaunch] RTCTREE_NAMESERVERS localhost:15005 localhost:15005
[rtmlaunch] SIMULATOR_NAME HiroNX(Robot)0
[rtmlaunch] check connection/activation
[rtmlaunch] check connection/activation
[rtmlaunch] Connected from localhost:15005/HiroNX(Robot)0.rtc:q
[rtmlaunch]             to localhost:15005/CollisionDetector0.rtc:qCurrent
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[rtmlaunch] Wait for  /localhost:15005/HrpsysSeqStateROSBridge0.rtc:rsangle   0 /30
[rtmlaunch] starting...  /home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/launch/nextage_ros_bridge_simulation.launch
[rtmlaunch] RTCTREE_NAMESERVERS localhost:15005 localhost:15005
[rtmlaunch] SIMULATOR_NAME Simulator
[rtmlaunch] Connected from localhost:15005/HiroNX(Robot)0.rtc:q
[rtmlaunch]             to localhost:15005/CollisionDetector0.rtc:qRef
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[rtmlaunch] Activate <- Inactive /localhost:15005/CollisionDetector0.rtc
[CollisionDetector0] onActivated(0)
[CollisionDetector0] set safe posture
[rtmlaunch] check connection/activation
configuration ORB with localhost:15005
[rtmlaunch] Wait for  /localhost:15005/ImageSensorROSBridge_HeadRight.rtc:timedImage   0 /30
[INFO] [WallTime: 1484856487.475611] [0.000000]   bodyinfo URL = file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_description/models/main.wrl
loading file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_description/models/main.wrl
Humanoid node
Joint node WAIST
  Segment node WAIST
  Joint node CHEST_JOINT0
    Segment node CHEST_JOINT0_Link
    Joint node HEAD_JOINT0
      Segment node HEAD_JOINT0_Link
      Joint node HEAD_JOINT1
        Segment node HEAD_JOINT1_Link
        VisionSensorCAMERA_HEAD_R
        VisionSensorCAMERA_HEAD_L
    Joint node RARM_JOINT0
      Segment node RARM_JOINT0_Link
      Joint node RARM_JOINT1
        Segment node RARM_JOINT1_Link
        Joint node RARM_JOINT2
          Segment node RARM_JOINT2_Link
          Joint node RARM_JOINT3
            Segment node RARM_JOINT3_Link
            Joint node RARM_JOINT4
              Segment node RARM_JOINT4_Link
              Joint node RARM_JOINT5
                Segment node RARM_JOINT5_Link
                VisionSensorCAMERA_HAND_R
    Joint node LARM_JOINT0
      Segment node LARM_JOINT0_Link
      Joint node LARM_JOINT1
        Segment node LARM_JOINT1_Link
        Joint node LARM_JOINT2
          Segment node LARM_JOINT2_Link
          Joint node LARM_JOINT3
            Segment node LARM_JOINT3_Link
            Joint node LARM_JOINT4
              Segment node LARM_JOINT4_Link
              Joint node LARM_JOINT5
                Segment node LARM_JOINT5_Link
                VisionSensorCAMERA_HAND_L
Node is inconvertible and removed from the scene graph
Node is inconvertible and removed from the scene graph
[ WARN] [1484856487.557958869]: [HrpsysSeqStateROSBridge] use_hrpsys_time
Warning: Mass is zero. <Model>HiroNX <Link>CHEST_JOINT0
Warning: Mass is zero. <Model>HiroNX <Link>WAIST
The model was successfully loaded !
[sensor_ros_bridge_connect.py] wait for RTCmanager : tork-kudu1
[sensor_ros_bridge_connect.py] wait for HiroNX(Robot)0 : <hrpsys.rtm.RTcomponent instance at 0x7f9608ed5cb0> ( timeout 0 < 10)
[sensor_ros_bridge_connect.py] findComps -> RobotHardware : <hrpsys.rtm.RTcomponent instance at 0x7f9608ed5cb0> isActive? = True
[sensor_ros_bridge_connect.py] simulation_mode : True
configuration ORB with localhost:15005
[hrpsys.py] waiting ModelLoader
[hrpsys.py] start hrpsys
[hrpsys.py] finding RTCManager and RobotHardware
[hrpsys.py]  waitForRTCManagerAndRoboHardware has renamed to waitForRTCManagerAndRoboHardware: Please update your code
[ INFO] [1484856487.662901432]: [HrpsysSeqStateROSBridge] @Initilize name : HrpsysSeqStateROSBridge0
cache found for /home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_description/models/main.wrl
Loading body customizer "/opt/ros/indigo/share/OpenHRP-3.1/customizer/libSampleCustomizer.so" for springJoint
cache found for /home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_description/models/main.wrl
duplicated sensor Id is specified(id = 0, name = CAMERA_HEAD_R)
duplicated sensor Id is specified(id = 1, name = CAMERA_HEAD_L)
duplicated sensor Id is specified(id = 2, name = CAMERA_HAND_R)
duplicated sensor Id is specified(id = 3, name = CAMERA_HAND_L)
[ INFO] [1484856487.675547018]: [HrpsysSeqStateROSBridge] Loaded HiroNX
[ INFO] [1484856487.675688503]: [HrpsysSeqStateROSBridge] @Initilize name : HrpsysSeqStateROSBridge0 done
[rtmlaunch] Wait for  /localhost:15005/HrpsysSeqStateROSBridge0.rtc:rsangle   1 /30
[rtmlaunch] Wait for  /localhost:15005/ImageSensorROSBridge_HeadRight.rtc:timedImage   1 /30
[sensor_ros_bridge_connect.py]   wait for  HrpsysSeqStateROSBridge0  :  <hrpsys.rtm.RTcomponent instance at 0x7f9608b0eef0>
[hrpsys.py] wait for RTCmanager : None
[hrpsys.py] wait for HiroNX(Robot)0 : <hrpsys.rtm.RTcomponent instance at 0x7fe0669fc098> ( timeout 0 < 10)
[hrpsys.py] findComps -> RobotHardware : <hrpsys.rtm.RTcomponent instance at 0x7fe0669fc098> isActive? = True
[hrpsys.py] simulation_mode : True
[hrpsys.py]  Hrpsys controller version info:
[hrpsys.py]    ms =  <hrpsys.rtm.RTCmanager instance at 0x7fe0669e92d8>
[hrpsys.py]    ref =  <RTM._objref_Manager instance at 0x7fe0669fcbd8>
[hrpsys.py]    version  =  315.10.1
[hrpsys.py]   bodyinfo URL = file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_description/models/main.wrl
cache found for file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_description/models/main.wrl
[hrpsys.py] no hrpsys components found running. creating now
[hrpsys.py]   eval : [self.seq, self.seq_svc, self.seq_version] = self.createComp("SequencePlayer","seq")
cache found for file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
[hrpsys.py] create Comp -> SequencePlayer : <hrpsys.rtm.RTcomponent instance at 0x7fe0669fc128> (315.10.1)
[hrpsys.py] create CompSvc -> SequencePlayer Service : <OpenHRP._objref_SequencePlayerService instance at 0x7fe066ac7dd0>
[hrpsys.py]   eval : [self.sh, self.sh_svc, self.sh_version] = self.createComp("StateHolder","sh")
[sh] onInitialize()
cache found for file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
[hrpsys.py] create Comp -> StateHolder : <hrpsys.rtm.RTcomponent instance at 0x7fe07e38e7e8> (315.10.1)
[hrpsys.py] create CompSvc -> StateHolder Service : <OpenHRP._objref_StateHolderService instance at 0x7fe066b6a3b0>
[hrpsys.py]   eval : [self.fk, self.fk_svc, self.fk_version] = self.createComp("ForwardKinematics","fk")
[fk] onInitialize()
cache found for file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
cache found for file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
[hrpsys.py] create Comp -> ForwardKinematics : <hrpsys.rtm.RTcomponent instance at 0x7fe066ac7bd8> (315.10.1)
[hrpsys.py] create CompSvc -> ForwardKinematics Service : <OpenHRP._objref_ForwardKinematicsService instance at 0x7fe066a09488>
[hrpsys.py]   eval : [self.ic, self.ic_svc, self.ic_version] = self.createComp("ImpedanceController","ic")
[ic] onInitialize()
cache found for file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
[ic] force sensor ports
[ic] End Effector [rarm]RARM_JOINT5 CHEST_JOINT0
[ic]   target = RARM_JOINT5, base = CHEST_JOINT0
[ic]   localPos =     [0,  0,  0][m]
[ic]   localR =     [1.40361e-09, 7.01805e-10,           1]
    [          1, 1.40361e-09, 7.01805e-10]
    [7.01805e-10,           1, 1.40361e-09]
[ic] End Effector [larm]LARM_JOINT5 CHEST_JOINT0
[ic]   target = LARM_JOINT5, base = CHEST_JOINT0
[ic]   localPos =     [0,  0,  0][m]
[ic]   localR =     [1.40361e-09, 7.01805e-10,           1]
    [          1, 1.40361e-09, 7.01805e-10]
    [7.01805e-10,           1, 1.40361e-09]
[ic] Add impedance params
[hrpsys.py] create Comp -> ImpedanceController : <hrpsys.rtm.RTcomponent instance at 0x7fe07e387bd8> (315.10.1)
[hrpsys.py] create CompSvc -> ImpedanceController Service : <OpenHRP._objref_ImpedanceControllerService instance at 0x7fe066ad3098>
[hrpsys.py]   eval : [self.el, self.el_svc, self.el_version] = self.createComp("SoftErrorLimiter","el")
;;
;; Could not open /dev/console for writing.
;;
open: Permission denied
[el] onInitialize()
cache found for file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
[INFO] [WallTime: 1484856488.813356] [0.000000] Connecting to RTM nameserver. host=localhost, port=15005, robotname=HiroNX(Robot)0
configuration ORB with localhost:15005
[hrpsys.py] waiting ModelLoader
[hrpsys.py] start hrpsys
[hrpsys.py] finding RTCManager and RobotHardware
[hrpsys.py]  waitForRTCManagerAndRoboHardware has renamed to waitForRTCManagerAndRoboHardware: Please update your code
[el] Do not load joint limit table
[hrpsys.py] create Comp -> SoftErrorLimiter : <hrpsys.rtm.RTcomponent instance at 0x7fe07e3966c8> (315.10.1)
[hrpsys.py] create CompSvc -> SoftErrorLimiter Service : <OpenHRP._objref_SoftErrorLimiterService instance at 0x7fe066ad3cb0>
[hrpsys.py]   eval : [self.sc, self.sc_svc, self.sc_version] = self.createComp("ServoController","sc")
[WARNING] sc: needs servo.devname property
[WARNING] sc: running in dummy mode
[hrpsys.py] create Comp -> ServoController : <hrpsys.rtm.RTcomponent instance at 0x7fe07e387638> (315.10.1)
[hrpsys.py] create CompSvc -> ServoController Service : <OpenHRP._objref_ServoControllerService instance at 0x7fe066ac7cb0>
[hrpsys.py]   eval : [self.log, self.log_svc, self.log_version] = self.createComp("DataLogger","log")
[log] onInitialize()
[hrpsys.py] create Comp -> DataLogger : <hrpsys.rtm.RTcomponent instance at 0x7fe07e396950> (315.10.1)
[hrpsys.py] create CompSvc -> DataLogger Service : <OpenHRP._objref_DataLoggerService instance at 0x7fe066ac7fc8>
[hrpsys.py]   eval : [self.rmfo, self.rmfo_svc, self.rmfo_version] = self.createComp("RemoveForceSensorLinkOffset","rmfo")
[rmfo] onInitialize()
cache found for file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
[hrpsys.py] create Comp -> RemoveForceSensorLinkOffset : <hrpsys.rtm.RTcomponent instance at 0x7fe066a09e60> (315.10.1)
[hrpsys.py] create CompSvc -> RemoveForceSensorLinkOffset Service : <OpenHRP._objref_RemoveForceSensorLinkOffsetService instance at 0x7fe07e38ee18>
[hrpsys.py] connecting hrpsys components
[rtm.py]    Connect sh.qOut - ic.qRef (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect ic.q - el.qRef (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect el.q - HGcontroller0.qIn (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect HGcontroller0.qOut - HiroNX(Robot)0.qRef (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect HiroNX(Robot)0.servoState - el.servoStateIn (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect HiroNX(Robot)0.q - sh.currentQIn (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect HiroNX(Robot)0.q - fk.q (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect sh.qOut - fk.qRef (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect seq.qRef - sh.qIn (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect seq.tqRef - sh.tqIn (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect seq.basePos - sh.basePosIn (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect seq.baseRpy - sh.baseRpyIn (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect seq.zmpRef - sh.zmpIn (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect seq.optionalData - sh.optionalDataIn (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect sh.basePosOut - seq.basePosInit (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect sh.basePosOut - fk.basePosRef (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect sh.baseRpyOut - seq.baseRpyInit (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect sh.baseRpyOut - fk.baseRpyRef (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect sh.qOut - seq.qInit (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect sh.zmpOut - seq.zmpRefInit (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect HiroNX(Robot)0.q - rmfo.qCurrent (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect HiroNX(Robot)0.q - ic.qCurrent (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect sh.basePosOut - ic.basePosIn (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect sh.baseRpyOut - ic.baseRpyIn (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[rtm.py]    Connect HiroNX(Robot)0.q - el.qCurrent (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[hrpsys.py] activating hrpsys components
[fk] onActivated(1000)
[ic] onActivated(1000)
[el] onActivated(1000)
[rmfo] onActivated(1000)
[hrpsys.py] setup hrpsys logger
[log] Log max length is set to 4000
[hrpsys.py]   setupLogger : record type = TimedDoubleSeq, name = q to HiroNX(Robot)0_q
[rtm.py]    Connect HiroNX(Robot)0.q - log.HiroNX(Robot)0_q (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[hrpsys.py]   setupLogger : record type = TimedDoubleSeq, name = tau to HiroNX(Robot)0_tau
[rtm.py]    Connect HiroNX(Robot)0.tau - log.HiroNX(Robot)0_tau (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[hrpsys.py] sensor names for DataLogger
[hrpsys.py]   setupLogger : record type = TimedDoubleSeq, name = qOut to sh_qOut
[rtm.py]    Connect sh.qOut - log.sh_qOut (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[hrpsys.py]   setupLogger : record type = TimedDoubleSeq, name = tqOut to sh_tqOut
[rtm.py]    Connect sh.tqOut - log.sh_tqOut (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[hrpsys.py]   setupLogger : record type = TimedPoint3D, name = basePosOut to sh_basePosOut
[rtm.py]    Connect sh.basePosOut - log.sh_basePosOut (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[hrpsys.py]   setupLogger : record type = TimedOrientation3D, name = baseRpyOut to sh_baseRpyOut
[rtm.py]    Connect sh.baseRpyOut - log.sh_baseRpyOut (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[hrpsys.py]   setupLogger : record type = TimedPoint3D, name = zmpOut to sh_zmpOut
[rtm.py]    Connect sh.zmpOut - log.sh_zmpOut (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[hrpsys.py]   setupLogger : record type = TimedDoubleSeq, name = q to ic_q
[rtm.py]    Connect ic.q - log.ic_q (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[hrpsys.py]   setupLogger : record type = TimedDoubleSeq, name = q to el_q
[rtm.py]    Connect el.q - log.el_q (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[hrpsys.py]   setupLogger : emergencySignal arleady exists in DataLogger
[rtm.py]    Connect HiroNX(Robot)0.emergencySignal - log.emergencySignal (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[hrpsys.py]   setupLogger : record type = TimedLongSeqSeq, name = servoState to HiroNX(Robot)0_servoState
[rtm.py]    Connect HiroNX(Robot)0.servoState - log.HiroNX(Robot)0_servoState (dataflow_type=Push, subscription_type=flush, bufferlength=1, push_rate=1000, push_policy=new)
[log] Log cleared
[hrpsys.py] setup joint groups for hrpsys controller
[addJointGroup] group name = torso
[addJointGroup] group name = head
[addJointGroup] group name = rarm
[addJointGroup] group name = larm
[hrpsys.py] initialized successfully
[addJointGroup] group name = torso
[addJointGroup] group name TORSO is already installed
[addJointGroup] group name = head
[addJointGroup] group name HEAD is already installed
[addJointGroup] group name = rarm
[addJointGroup] group name RARM is already installed
[addJointGroup] group name = larm
[addJointGroup] group name LARM is already installed
[rtmlaunch] Wait for  /localhost:15005/HrpsysSeqStateROSBridge0.rtc:rsangle   2 /30
[rtmlaunch] Connected from localhost:15005/HiroNX(Robot)0.rtc:q
[rtmlaunch]             to localhost:15005/HrpsysSeqStateROSBridge0.rtc:rsangle
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[rtmlaunch] Connected from localhost:15005/HiroNX(Robot)0.rtc:tau
[rtmlaunch]             to localhost:15005/HrpsysSeqStateROSBridge0.rtc:rstorque
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[rtmlaunch] Wait for  /localhost:15005/sh.rtc:StateHolderService   0 /30
[rtmlaunch] Connected from localhost:15005/StateHolderServiceROSBridge.rtc:StateHolderService
[rtmlaunch]             to localhost:15005/sh.rtc:StateHolderService
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[rtmlaunch] Connected from localhost:15005/sh.rtc:baseTformOut
[rtmlaunch]             to localhost:15005/HrpsysSeqStateROSBridge0.rtc:baseTform
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[ERROR] [1484856489.526275232]: Target Node visual1/node_joint0_axis0 NOT found!!!

[ WARN] [1484856489.530634150]: could not find binding for axis: kmodel1/jointsid1000/axis0, axis0

[ WARN] [1484856489.533575796]: could not find binding for axis: kmodel1/jointsid1000/axis0, axis0

[rtmlaunch] Wait for  /localhost:15005/ImageSensorROSBridge_HeadRight.rtc:timedImage   2 /30
[rtmlaunch] [ERROR] Could not Connect ( /localhost:15005/HiroNX(Robot)0.rtc:CAMERA_HEAD_R , /localhost:15005/ImageSensorROSBridge_HeadRight.rtc:timedImage ):  'NoneType' object has no attribute 'connections'
[rtmlaunch] Activate <- Inactive /localhost:15005/ImageSensorROSBridge_HeadRight.rtc
[rtmlaunch] Activate <- Inactive /localhost:15005/ImageSensorROSBridge_HeadLeft.rtc
[rtmlaunch] Activate <- Inactive /localhost:15005/ImageSensorROSBridge_HandRight.rtc
[rtmlaunch] Activate <- Inactive /localhost:15005/ImageSensorROSBridge_HandLeft.rtc
[sensor_ros_bridge_connect.py]   wait for 'sh' :  <hrpsys.rtm.RTcomponent instance at 0x7f9608b0ea70>
[sensor_ros_bridge_connect.py]   bodyinfo URL = file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_description/models/main.wrl
cache found for file:///home/rospasta/cws_hironxo/src/tork-a/rtmros_nextage/nextage_description/models/main.wrl
[rtmlaunch] Connected from localhost:15005/sh.rtc:qOut
[rtmlaunch]             to localhost:15005/HrpsysSeqStateROSBridge0.rtc:mcangle
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[rtmlaunch] Connected from localhost:15005/HrpsysSeqStateROSBridge0.rtc:SequencePlayerService
[rtmlaunch]             to localhost:15005/seq.rtc:SequencePlayerService
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[ WARN] [1484856489.640173297]: could not find openrave joint WAIST!

[rtmlaunch] Connected from localhost:15005/HrpsysJointTrajectoryBridge0.rtc:SequencePlayerService
[rtmlaunch]             to localhost:15005/seq.rtc:SequencePlayerService
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[rtmlaunch] Connected from localhost:15005/DataLoggerServiceROSBridge.rtc:DataLoggerService
[rtmlaunch]             to localhost:15005/log.rtc:DataLoggerService
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[rtmlaunch] Connected from localhost:15005/SequencePlayerServiceROSBridge.rtc:SequencePlayerService
[rtmlaunch]             to localhost:15005/seq.rtc:SequencePlayerService
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[ERROR] [1484856489.647717416]: Robot semantic description not found. Did you forget to define or remap '/robot_description_semantic'?
[ INFO] [1484856489.647808241]: Loading robot model 'HiroNX'...
[ INFO] [1484856489.647857007]: No root/virtual joint specified in SRDF. Assuming fixed joint
[rtmlaunch] Connected from localhost:15005/ForwardKinematicsServiceROSBridge.rtc:ForwardKinematicsService
[rtmlaunch]             to localhost:15005/fk.rtc:ForwardKinematicsService
[rtmlaunch]           with {'id': '', 'properties': {'dataport.publisher.push_policy': 'all', 'dataport.subscription_type': 'new'}, 'name': None, 'verbose': False}
[rtmlaunch] Activate <- Inactive /localhost:15005/HrpsysSeqStateROSBridge0.rtc
[rtmlaunch] Activate <- Inactive /localhost:15005/HrpsysJointTrajectoryBridge0.rtc
[rtmlaunch] Activate <- Inactive /localhost:15005/DataLoggerServiceROSBridge.rtc
[rtmlaunch] Activate <- Inactive /localhost:15005/SequencePlayerServiceROSBridge.rtc
[ INFO] [1484856489.653158712, 2.944999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 0[Hz] (exec 1 Hz, dropped 0)
[rtmlaunch] Activate <- Inactive /localhost:15005/ForwardKinematicsServiceROSBridge.rtc
[rtmlaunch] Activate <- Inactive /localhost:15005/StateHolderServiceROSBridge.rtc
[ INFO] [1484856489.658401983, 2.984999999]: ADD_GROUP: larm (/larm_controller)
[ INFO] [1484856489.658475138, 2.984999999]:     JOINT: LARM_JOINT0 LARM_JOINT1 LARM_JOINT2 LARM_JOINT3 LARM_JOINT4 LARM_JOINT5
[addJointGroup] group name = larm
[addJointGroup] group name LARM is already installed
[sensor_ros_bridge_connect-11] process has finished cleanly
log file: /home/rospasta/.ros/log/fcc43aec-de82-11e6-ab69-80fa5b08d1d8/sensor_ros_bridge_connect-11*.log
[ INFO] [1484856489.728644044, 3.059999999]: ADD_GROUP: rarm (/rarm_controller)
[ INFO] [1484856489.728705236, 3.059999999]:     JOINT: RARM_JOINT0 RARM_JOINT1 RARM_JOINT2 RARM_JOINT3 RARM_JOINT4 RARM_JOINT5
[addJointGroup] group name = rarm
[addJointGroup] group name RARM is already installed
[ INFO] [1484856489.771671395, 3.099999999]: ADD_GROUP: head (/head_controller)
[ INFO] [1484856489.771712871, 3.099999999]:     JOINT: HEAD_JOINT0 HEAD_JOINT1
[addJointGroup] group name = head
[addJointGroup] group name HEAD is already installed
[ INFO] [1484856489.808272047, 3.139999999]: ADD_GROUP: torso (/torso_controller)
[ INFO] [1484856489.808310562, 3.139999999]:     JOINT: CHEST_JOINT0
[addJointGroup] group name = torso
[addJointGroup] group name TORSO is already installed
[hrpsys.py] wait for RTCmanager : None
[hrpsys.py] wait for HiroNX(Robot)0 : <hrpsys.rtm.RTcomponent instance at 0x7fdd195c4f80> ( timeout 0 < 10)
[hrpsys.py] findComps -> RobotHardware : <hrpsys.rtm.RTcomponent instance at 0x7fdd195c4f80> isActive? = True
[hrpsys.py] simulation_mode : True
[hrpsys.py]  Hrpsys controller version info:
[hrpsys.py]    ms =  <hrpsys.rtm.RTCmanager instance at 0x7fdd195c4d88>
[hrpsys.py]    ref =  <RTM._objref_Manager instance at 0x7fdd195c4638>
[ INFO] [1484856489.842685769, 3.174999999]: ADD_GROUP: larm_torso (/larm_torso_controller)
[ INFO] [1484856489.842769534, 3.174999999]:     JOINT: LARM_JOINT0 LARM_JOINT1 LARM_JOINT2 LARM_JOINT3 LARM_JOINT4 LARM_JOINT5 CHEST_JOINT0
[addJointGroup] group name = larm_torso
[ INFO] [1484856489.891771888, 3.219999999]: ADD_GROUP: rarm_torso (/rarm_torso_controller)
[ INFO] [1484856489.891846617, 3.219999999]:     JOINT: RARM_JOINT0 RARM_JOINT1 RARM_JOINT2 RARM_JOINT3 RARM_JOINT4 RARM_JOINT5 CHEST_JOINT0
[addJointGroup] group name = rarm_torso
[hrpsys.py]    version  =  315.10.1
[hrpsys.py] hrpsys components are already created and running
[hrpsys.py] [self.seq, self.seq_svc, self.seq_version] = self.findComp("SequencePlayer","seq",0,True)
[hrpsys.py]  find Comp    : seq = <hrpsys.rtm.RTcomponent instance at 0x7fdd19deb368> (315.10.1)
[hrpsys.py]  find CompSvc : seq_svc = <OpenHRP._objref_SequencePlayerService instance at 0x7fdd193e2680>
[hrpsys.py] [self.sh, self.sh_svc, self.sh_version] = self.findComp("StateHolder","sh",0,True)
[hrpsys.py]  find Comp    : sh = <hrpsys.rtm.RTcomponent instance at 0x7fdd195d7e60> (315.10.1)
[hrpsys.py]  find CompSvc : sh_svc = <OpenHRP._objref_StateHolderService instance at 0x7fdd18a79758>
[hrpsys.py] [self.fk, self.fk_svc, self.fk_version] = self.findComp("ForwardKinematics","fk",0,True)
[hrpsys.py]  find Comp    : fk = <hrpsys.rtm.RTcomponent instance at 0x7fdd1a190878> (315.10.1)
[hrpsys.py]  find CompSvc : fk_svc = <OpenHRP._objref_ForwardKinematicsService instance at 0x7fdd1949b5f0>
[hrpsys.py] [self.ic, self.ic_svc, self.ic_version] = self.findComp("ImpedanceController","ic",0,True)
[hrpsys.py]  find Comp    : ic = <hrpsys.rtm.RTcomponent instance at 0x7fdd19dd0cf8> (315.10.1)
[hrpsys.py]  find CompSvc : ic_svc = <OpenHRP._objref_ImpedanceControllerService instance at 0x7fdd195dc3f8>
[hrpsys.py] [self.el, self.el_svc, self.el_version] = self.findComp("SoftErrorLimiter","el",0,True)
[hrpsys.py]  find Comp    : el = <hrpsys.rtm.RTcomponent instance at 0x7fdd194b1b00> (315.10.1)
[hrpsys.py]  find CompSvc : el_svc = <OpenHRP._objref_SoftErrorLimiterService instance at 0x7fdd195dc488>
[hrpsys.py] [self.sc, self.sc_svc, self.sc_version] = self.findComp("ServoController","sc",0,True)
[hrpsys.py]  find Comp    : sc = <hrpsys.rtm.RTcomponent instance at 0x7fdd195c48c0> (315.10.1)
[hrpsys.py]  find CompSvc : sc_svc = <OpenHRP._objref_ServoControllerService instance at 0x7fdd195d7908>
[hrpsys.py] [self.log, self.log_svc, self.log_version] = self.findComp("DataLogger","log",0,True)
[hrpsys.py]  find Comp    : log = <hrpsys.rtm.RTcomponent instance at 0x7fdd194c2290> (315.10.1)
[hrpsys.py]  find CompSvc : log_svc = <OpenHRP._objref_DataLoggerService instance at 0x7fdd1959c830>
[hrpsys.py] [self.rmfo, self.rmfo_svc, self.rmfo_version] = self.findComp("RemoveForceSensorLinkOffset","rmfo",0,True)
[hrpsys.py]  find Comp    : rmfo = <hrpsys.rtm.RTcomponent instance at 0x7fdd195e95f0> (315.10.1)
[hrpsys.py]  find CompSvc : rmfo_svc = <OpenHRP._objref_RemoveForceSensorLinkOffsetService instance at 0x7fdd18a75ef0>
[hrpsys.py] setup joint groups for hrpsys controller
[addJointGroup] group name = torso
[addJointGroup] group name TORSO is already installed
[addJointGroup] group name = head
[addJointGroup] group name HEAD is already installed
[addJointGroup] group name = rarm
[addJointGroup] group name RARM is already installed
[addJointGroup] group name = larm
[addJointGroup] group name LARM is already installed
[hrpsys.py] initialized successfully
[addJointGroup] group name = torso
[addJointGroup] group name TORSO is already installed
[addJointGroup] group name = head
[addJointGroup] group name HEAD is already installed
[addJointGroup] group name = rarm
[addJointGroup] group name RARM is already installed
[addJointGroup] group name = larm
[addJointGroup] group name LARM is already installed
[INFO] [WallTime: 1484856490.380029] [3.710000] Joint names; Torso: ['CHEST_JOINT0']
        Head: ['HEAD_JOINT0', 'HEAD_JOINT1']
        L-Arm: ['LARM_JOINT0', 'LARM_JOINT1', 'LARM_JOINT2', 'LARM_JOINT3', 'LARM_JOINT4', 'LARM_JOINT5']
        R-Arm: ['RARM_JOINT0', 'RARM_JOINT1', 'RARM_JOINT2', 'RARM_JOINT3', 'RARM_JOINT4', 'RARM_JOINT5']
[WARN] [WallTime: 1484856490.382292] [3.715000] Moveit is not started yet, if you want to use MoveIt!
Make sure you've launched MoveGroup (e.g. by launching moveit_planning_execution.launch)
[hrpsys_py-4] process has finished cleanly
log file: /home/rospasta/.ros/log/fcc43aec-de82-11e6-ab69-80fa5b08d1d8/hrpsys_py-4*.log
Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rqt_console/message_proxy_model.py", line 118, in handle_highlight_filters_changed
    self.dataChanged.emit(self.index(0, 0), self.index(self.rowCount() - 1, self.columnCount() - 1), [])
TypeError: QAbstractItemModel.dataChanged[QModelIndex, QModelIndex] signal has 2 argument(s) but 3 provided
[ INFO] [1484856490.659520935, 3.989999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 209[Hz] (exec 1765 Hz, dropped 209)
Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rqt_console/message_proxy_model.py", line 118, in handle_highlight_filters_changed
    self.dataChanged.emit(self.index(0, 0), self.index(self.rowCount() - 1, self.columnCount() - 1), [])
TypeError: QAbstractItemModel.dataChanged[QModelIndex, QModelIndex] signal has 2 argument(s) but 3 provided
[ INFO] [1484856491.666849513, 4.999999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 203[Hz] (exec 1785 Hz, dropped 203)
[INFO] [WallTime: 1484856491.682555] [5.025000] check 4 collision status, 0.796020 Hz
[ INFO] [1484856492.669410264, 6.014999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1795 Hz, dropped 202)
[ INFO] [1484856493.669688484, 7.019999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1748 Hz, dropped 201)
[ INFO] [1484856494.678145932, 8.029999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1798 Hz, dropped 202)
[ INFO] [1484856495.678822748, 9.040000000]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1791 Hz, dropped 202)
[INFO] [WallTime: 1484856496.676598] [10.040000] check 4 collision status, 0.797607 Hz
[ INFO] [1484856496.679869640, 10.045000000]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1776 Hz, dropped 201)
[rtmlaunch] check connection/activation
[ INFO] [1484856497.680295712, 11.050000000]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1787 Hz, dropped 201)
[ INFO] [1484856498.680686827, 12.055000000]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1794 Hz, dropped 201)
[ INFO] [1484856499.682032651, 13.060000000]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1785 Hz, dropped 201)
[rtmlaunch] check connection/activation
[rtmlaunch] [ERROR] Could not Connect ( /localhost:15005/HiroNX(Robot)0.rtc:CAMERA_HEAD_R , /localhost:15005/ImageSensorROSBridge_HeadRight.rtc:timedImage ):  'NoneType' object has no attribute 'connections' [rtmlaunch] check connection/activation[ INFO] [1484856500.686626627, 14.070000000]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1792 Hz, dropped 202)


[INFO] [WallTime: 1484856504.224291] [17.610000] check 4 collision status, 0.529101 Hz
[ INFO] [1484856505.223687808, 18.625000000]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 210[Hz] (exec 1794 Hz, dropped 210)
[ INFO] [1484856506.224996280, 19.630000000]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1787 Hz, dropped 201)
[ INFO] [1484856507.227300108, 20.635000000]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1792 Hz, dropped 201)
[rtmlaunch] check connection/activation
[ INFO] [1484856508.230577239, 21.644999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1788 Hz, dropped 202)
[INFO] [WallTime: 1484856509.191455] [22.605000] check 4 collision status, 0.799201 Hz
[ INFO] [1484856509.231303997, 22.649999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1786 Hz, dropped 201)
[ INFO] [1484856510.236939883, 23.654999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1790 Hz, dropped 201)
[ INFO] [1484856511.241437409, 24.669999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 203[Hz] (exec 1800 Hz, dropped 203)
[ INFO] [1484856512.246203655, 25.674999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1799 Hz, dropped 201)
[ INFO] [1484856513.249126327, 26.684999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1794 Hz, dropped 202)
[INFO] [WallTime: 1484856514.184669] [27.625000] check 4 collision status, 0.796813 Hz
[ INFO] [1484856514.252014950, 27.689999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1790 Hz, dropped 201)
[rtmlaunch] check connection/activation
[rtmlaunch] [ERROR] Could not Connect ( /localhost:15005/HiroNX(Robot)0.rtc:CAMERA_HEAD_R , /localhost:15005/ImageSensorROSBridge_HeadRight.rtc:timedImage ):  'NoneType' object has no attribute 'connections'
[rtmlaunch] check connection/activation
[ INFO] [1484856515.254709573, 28.699999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1788 Hz, dropped 202)
[ INFO] [1484856516.263357267, 29.709999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1803 Hz, dropped 202)
[ INFO] [1484856517.267568499, 30.719999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1782 Hz, dropped 202)
[rtmlaunch] check connection/activation
[ INFO] [1484856518.271197254, 31.729999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1798 Hz, dropped 202)
[INFO] [WallTime: 1484856519.185977] [32.645000] check 4 collision status, 0.796813 Hz
[ INFO] [1484856519.272574623, 32.734999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1790 Hz, dropped 201)
[ INFO] [1484856520.277136985, 33.744999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1798 Hz, dropped 202)
[ INFO] [1484856521.280965765, 34.749999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1794 Hz, dropped 201)
[ INFO] [1484856522.282773888, 35.759999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 202[Hz] (exec 1796 Hz, dropped 202)
[ INFO] [1484856523.283117067, 36.765000000]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1780 Hz, dropped 201)
[INFO] [WallTime: 1484856524.183140] [37.665000] check 4 collision status, 0.796813 Hz
[ INFO] [1484856524.287469469, 37.770000000]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1800 Hz, dropped 201)
[rtmlaunch] check connection/activation
[rtmlaunch] [ERROR] Could not Connect ( /localhost:15005/HiroNX(Robot)0.rtc:CAMERA_HEAD_R , /localhost:15005/ImageSensorROSBridge_HeadRight.rtc:timedImage ):  'NoneType' object has no attribute 'connections'
```

--- a/nextage_ros_bridge/conf/nextage.xml.in
+++ b/nextage_ros_bridge/conf/nextage.xml.in
@@ -21,6 +21,10 @@
    <property name="inport" value="ddqRef:JOINT_ACCELERATION"/>
    <property name="outport" value="q:JOINT_VALUE"/>
    <property name="outport" value="tau:JOINT_TORQUE"/>
+   <property name="outport" value="CAMERA_HEAD_R:CAMERA_HEAD_R:VISION_SENSOR"/>
+   <property name="outport" value="CAMERA_HEAD_L:CAMERA_HEAD_L:VISION_SENSOR"/>
+   <property name="outport" value="CAMERA_HAND_R:CAMERA_HAND_R:VISION_SENSOR"/>
+   <property name="outport" value="CAMERA_HAND_L:CAMERA_HAND_L:VISION_SENSOR"/>
    <property name="WAIST.NumOfAABB" value="1"/>
    <property name="WAIST.translation" value="0  0  0"/>
    <property name="WAIST.rotation" value="1 0 0 0"/>

--- a/nextage_ros_bridge/conf/nextage_nosim.xml.in
+++ b/nextage_ros_bridge/conf/nextage_nosim.xml.in
@@ -21,6 +21,10 @@
    <property name="inport" value="ddqRef:JOINT_ACCELERATION"/>
    <property name="outport" value="q:JOINT_VALUE"/>
    <property name="outport" value="tau:JOINT_TORQUE"/>
+   <property name="outport" value="CAMERA_HEAD_R:CAMERA_HEAD_R:VISION_SENSOR"/>
+   <property name="outport" value="CAMERA_HEAD_L:CAMERA_HEAD_L:VISION_SENSOR"/>
+   <property name="outport" value="CAMERA_HAND_R:CAMERA_HAND_R:VISION_SENSOR"/>
+   <property name="outport" value="CAMERA_HAND_L:CAMERA_HAND_L:VISION_SENSOR"/>
    <property name="WAIST.NumOfAABB" value="1"/>
    <property name="WAIST.translation" value="0  0  0"/>
    <property name="WAIST.rotation" value="1 0 0 0"/>

